extensions/cornerstone/src/initContextMenu.ts

```diff
@@ -0,0 +1,132 @@
+import { eventTarget, Types, EVENTS } from '@cornerstonejs/core';
+import { Enums } from '@cornerstonejs/tools';
+import { setEnabledElement } from './state';
+
+const cs3DToolsEvents = Enums.Events;
+
+const showContextMenuDefault: Types.CommandCustomization = {
+  commands: [
+    {
+      commandName: 'showViewerContextMenu',
```

Comment:
There is `showContextMenu` and `showViewerContextMenu` what is the deal with these? Is there any non-viewer context menu?

Reply:
Cleaned that up - yes, there are differences between the menu commands.  This one is cornersone viewport specific, and there might be others which apply other viewport types or non-viewpor related context menus.